src/main/java/com/example/Restaurant_Management_System_REST_API/model/entity/Customer.java

```diff
@@ -0,0 +1,73 @@
+package com.example.Restaurant_Management_System_REST_API.model.entity;
+
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+@Entity
+@Table(name = "application_users")
```

Comment:
Does name "user" is not sufficient enough?

Reply:
nope, I remember from my courses that "user" is an internal SQL word and it can be problematic to use this reserved word for table or column name.

Reply:
Yes, we did, Bartek :) That's why I mention it, because you could just name it "users" if you want to follow a plural convention of naming tables :)